check_msid_status.py

```diff
@@ -229,30 +209,6 @@ def check_status_letter(msid, val, vdict):
                     return 'CAUTION'
             else:
                 return 'GREEN'
-    
-    elif msid in ['PLINE03T', 'PLINE04T']:
-        if val >= 42.5:
-            return 'GREEN'
-        elif val < 40:
-            return 'WARNING'
-        elif val < 42.5:
-            return 'CAUION'
-    
-    elif msid == 'AACCCDPT':
-        if val >= 0:
-            return 'WARNING'
-        elif val > -17:
-            return 'CAUTION'
-        elif val < -21.5:
-            return 'CAUTION'
-        else:
-            return 'GREEN'
-        
-    elif msid == '3LDRTNO':
-        if val > 0:
-            return 'GREEN'
-        else:
-            return 'WARNING'
 
```

Comment:
The MSIDs 'PLINE03T', 'PLINE04T', 'AACCCDPT', '3LDRTNO' are all numerically valued MSIDs which have limits stored in the CSH_limit_table.json file. Therefore they do not need to be part of the edge case function.

Reply:
ok